pkg/cmd/cli/cmd/debug.go

```diff
@@ -390,6 +394,21 @@ func (o *DebugOptions) Debug() error {
 	})
 }
 
+func (o *DebugOptions) getContainerImageCommand(container *kapi.Container) ([]string, error) {
+	image := container.Image[strings.LastIndex(container.Image, "/")+1:]
+	name, id, ok := imageapi.SplitImageStreamImage(image)
+	if !ok {
+		return nil, errors.New("container image did not contain an id")
+	}
+	isimage, err := o.Client.ImageStreamImages(o.Attach.Pod.Namespace).Get(name, id)
```

Comment:
This was not what I meant.  I meant imagestreamimport, because that handles images that are on the cluster and images that are off, while this will fail for an image created with `oc run`.